pdc/apps/common/filters.py

```diff
@@ -203,22 +202,6 @@ def qs(self):
                     else:
                         qs = filter_.filter(qs, value)
 
-            if self._meta.order_by:
-                order_field = self.form.fields[self.order_by_field]
-                data = self.form[self.order_by_field].data
-                ordered_value = None
-                try:
-                    ordered_value = order_field.clean(data)
-                except forms.ValidationError:
-                    pass
-
-                if ordered_value in EMPTY_VALUES and self.strict:
-                    ordered_value = \
-                        self.form.fields[self.order_by_field].choices[0][0]
-
-                if ordered_value:
-                    qs = qs.order_by(*self.get_order_by(ordered_value))
-
```

Comment:
Why remove these codes, is that because it is never used?

Reply:
Yup, the commit is "Remove dead code".

Reply:
cool